src/components/approval-type/ApprovalType.vue

```diff
@@ -32,7 +32,7 @@
           </v-form>
           <!-- REGISTRAR section -->
           <v-radio v-if="!isCourtOrderOnly"
-            id="registrar-radio" class="mb-0 pt-2"
+            id="registrar-radio" class="mb-n5 pt-2"
```

Comment:
Remove bottom padding

Reply:
I think v-radio has a prop to remove extra bottom whitespace... Auto messages or something.

Reply:
It's OK like this.